core/src/main/scala/model/Metric.scala

```diff
@@ -18,6 +18,46 @@ package freestyle
 package opscenter
 package model
 
-case class Metric[T](metric: String, microservice: String, node: String, value: T, date: Long) {
-  override def toString(): String = s"$microservice.$node $date $metric $value"
+import freestyle.opscenter.protobuf.metric.{ListMetricsProto, MetricProto}
+
+case class Metric(metric: String, microservice: String, node: String, value: Float, date: Long) {
```

Comment:
We should model the `date` as joda-time [DateTime](http://joda-time.sourceforge.net/apidocs/org/joda/time/DateTime.html).

Reply:
Created issue for this #15